localstack/utils/aws/aws_stack.py

```diff
@@ -840,3 +840,8 @@ def kinesis_get_latest_records(stream_name, shard_id, count=10, env=None):
         while len(result) > count:
             result.pop(0)
     return result
+
+
+def get_lambda_name_from_arn(arn):
```

Comment:
We can remove this function and use `lambda_function_name(..)` in this file instead.